python/demo/gsoc2022/demo_sbc.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (C) 2022 Michele Castriotta
+#
+# ====================================================
+# Demo: scattering of an electromagnetic field by an
```

Comment:
The acronym in the filename sbc is not commonly known.

Reply:
I changed the name in "demo_scattering_boundary_conditions.py"